forter/forter-magento2
# Add support for Beneficiaries field and Gift Cards

### Goals
- When a gift card is purchased, the gift card recipient's first name, last name, email address and message to the recipient will be added to the Beneficiary object within the Item object sent to Forter
- If an order contains a gift card, the PersonalDetails object will be updated to contain the gift card recipient's first name, last name and email
- If an order contains multiple gift cards, the PersonalDetails object will be updated with the recipient's data from the first gift card in the order

Forter Note:
- Forter wanted to make sure that, for physical items, the beneficiary details will still be included within the item object (name and address of the recipient, email and phone only if entered for shipping).

### Testing Instructions
- Place a test order for each of the following scenarios and ensure the data in Forter is formatted as outlined in the Goals above:
-- An order with a physical shipped good
-- An order with only a Gift Card
-- An order with multiple Gift Cards
-- An order with a physical shipped good and a gift card
-- An order with a physical shipped good and multiple gift cards
- Ensure the updated module works with both Magento Commerce and Magento Open Source (Which does not have gift card functionality)

### Tested On
- Development instance
